kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ProxyReconcilerTest.java

```diff
@@ -94,7 +94,7 @@ void successfulInitialReconciliationShouldResultInReadyTrueCondition() {
 
         // Then
         assertThat(updateControl.isPatchStatus()).isTrue();
-        var statusAssert = assertThat(updateControl.getResource()).isNotNull()
+        var statusAssert = assertThat(updateControl.getResource()).get()
```

Comment:
Should we also add `.isNotEmpty()` ?

```suggestion
        var statusAssert = assertThat(updateControl.getResource()).isNotEmpty().get()
```